.travis.yml

```diff
@@ -1,13 +1,8 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
```

Comment:
Why remove all php versions? Surely we can test with 5.5 and 5.6 and hhvm at least.. And I'd argue we should still test for 5.3/5.4 but then run a `composer require symfony/symfony:^2.2 --no-update` before the `composer update` so that we don't test with the latest symfony.